articles/libraries/auth0-swift/index.md

```diff
@@ -77,7 +79,30 @@ Auth0
    }
 ```
 
-#### Passwordless Login
+### Passwordless Login
+
+Logging in with a Passwordless is slightly different. Passwordless can be done via email or via SMS, and either by sending the user a code, or sending them a link which contains a code. 
+
+#### Passwordless Parameters
+
+As you can see, Passwordless authentication can be started with a variety of different parameters. 
+
+```
+.startPasswordless(email/phoneNumber: String, type: String, connection: String)
+```
+
+- **Parameter One** - Either the parameter `email` or `phoneNumber`, depending on which you intend to use. The value of either should be a string.
+- **Parameter Two** - The parameter `type`, its value should be either `.Code` or `.iOSLink`. The default is `.Code` (a code is sent to the user, then they input it in a secondary screen), but if you have iOS Universal Links configured, you can use `.iOSLink`.
+- **Parameter Three** - The parameter `connection`, its value should be the name of the connection, defaults to `sms`.
+
+
+#### How Passwordless Works
```

Comment:
It might make more sense to talk about how passwordless works before demonstrating the passwordless methods? Would also remove 'at the heart of things' feels out of place.